wikitemplate-ios.md

```diff
@@ -108,47 +108,49 @@
+- [ ] Turning HTTPS Everywhere off or shields off both disable the redirect to `https://https-everywhere.badssl.com/`
+- [ ] Check that block ad and unblock ad works on `http://slashdot.org`
 - [ ] Check that toggling to blocking and allow ads works as expected
-- [ ] Verify that clicking through a cert error in https://badssl.com/ works
-- [ ] Verify that Safe Browsing works (https://www.raisegame.com/)
-- [ ] Enable block script globally from settings, Visit https://twitter.com/, nothing should load. Tap on Shields and disable block script, the page should load properly
-- [ ] Verify that preferences default Brave Shields settings take effect on pages with no site settings
+- [ ] Verify that tapping through a cert error in `https://badssl.com` works
+- [ ] Verify that Safe Browsing works `https://www.raisegame.com`
+- [ ] Enable `Block Scripts` globally from `Settings`.  Visit `https://twitter.com`, nothing should load. Tap on `Shields` and disable `Block Scripts`; the page should load properly.
+- [ ] Verify that default Brave Shields settings take effect on pages with no site settings
+
 ### Fingerprint Tests
-  - [ ] Verify that turning on fingerprinting protection in preferences shows 3 fingerprint blocked at https://browserleaks.com/canvas . Verify that turning it off in the Bravery menu shows 0 fingerprints blocked
-  - [ ] Verify that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on
-  - [ ] Verify that https://diafygi.github.io/webrtc-ips/ doesn't leak IP address when `Block all fingerprinting protection` is on
+
+  - [ ] Verify that turning on fingerprinting protection in preferences shows `3 fingerprints blocked` at `https://browserleaks.com/canvas`. Verify that turning it off in the Bravery menu shows `0 fingerprints blocked`.
+  - [ ] Verify that the audio fingerprint is blocked at `https://audiofingerprint.openwpm.com` when fingerprinting protection is on
+  - [ ] Verify that `https://diafygi.github.io/webrtc-ips/` doesn't leak IP address when `Block all fingerprinting protection` is on
 
 ## Content tests
 
-- [ ] Go to https://feedly.com/ and make sure that the password can be saved.  Make sure the saved password is auto-populated when you visit the site again
-- [ ] Open an email on http://mail.google.com/ or `inbox.google.com` and click on a link. Make sure it works
-- [ ] Verify that PDF is loaded over https at https://basicattentiontoken.org/BasicAttentionTokenWhitePaper-4.pdf
-- [ ] Verify that PDF is loaded over HTTP at http://www.pdf995.com/samples/pdf.pdf
-- [ ] Verify that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run)
-- [ ] Verify that search results from https://startpage.com/ open in a new tab (due to target being _blank_)
+- [ ] Go to `https://www.gmail.com` and make sure that the password can be saved.  Make sure the saved password is auto-populated when you visit the site again.
```

Comment:
Any reason to change this to Gmail? We had issues before where the prompt wasn't shown on certain devices. Prefer we keep it as is

Reply:
Just "flow state/convenience" when testing, but that's not a great reason, especially due to the issue you mentioned above.  Will revert.